### ISSUE TITLE
Labels vs ids for One way/turn restriction confidence conflict

### ISSUE THREAD

**Issue by CloCkWeRX** [Telenav/improveosm.org#3]
You have

```
<div class="tel_displayInline"><input id="C1" type="checkbox" checked="checked"><label for="C1">Super</label></div>
```

and

```
<div class="tel_displayInline twoPerRow"><input id="C1" type="checkbox" checked="checked"><label for="C1">High</label></div>
```

Meaning a click on the "Turn restriction- high" control interacts with the "One way - Super" radio.

![image](https://cloud.githubusercontent.com/assets/365751/20294206/5f43a5b4-ab4c-11e6-8e5e-34cbbf1390f0.png)

**Comment by tudorg-telenav** [comment]
@andreic-telenav please have a POJO retain a mapping of all the HTML ids over the strings expected by the API

**Comment by tudorg-telenav** [comment]
@andreic-telenav commited a fix for this in 6c2ffcadf7ef088718b5df10bba298fd40cefe6e
